src/main/java/org/broadinstitute/consent/http/models/DataUse.java

```diff
@@ -1,79 +1,44 @@
 package org.broadinstitute.consent.http.models;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
-import com.google.common.base.Objects;
 import com.google.gson.GsonBuilder;
 import java.util.List;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
 
-@SuppressWarnings({"unused", "SameParameterValue", "WeakerAccess"})
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class DataUse {
```

Comment:
[General Consent question]

Does Consent have the capability to auto-generate these models as part of your API spec definition?

Reply:
Actually, no, it doesn't, which reminds me that I need to update the swagger models to match this, thanks for the reminder!

Reply:
Turns out that `DataUse.yaml` is not affected by these changes, but does need to be rewritten. I'll tackle that separately.